libpromises/Makefile.am

```diff
@@ -216,5 +216,5 @@ CLEANFILES = *.gcno *.gcda $(BUILT_SOURCES)
 #
 MOSTLYCLEANFILES = *~ *.orig *.rej
 
-bootstrap.inc:
+bootstrap.inc: failsafe.cf
```

Comment:
Heh, a lot simpler than my solution: https://github.com/cfengine/core/commit/e51703ae4d9bc1971c5316aa2abe81ed09b22e82. I think this should be ok, as long as tar.gz keeps timestamps intact, which I believe it does.